projector-plugin/src/main/kotlin/org/jetbrains/projector/plugin/ui/ProjectorStatusWidget.kt

```diff
@@ -96,23 +91,45 @@ class ProjectorStatusWidget(project: Project)
     }
   }
 
-  private fun updateText() = "Projector"
+  private fun updateText() = "Projector" //  (${clientsCount()})"
 
   private fun updateTooltip(): String {
     return when {
       isActivationNeeded() -> "Activation is needed"
-      isProjectorRunning() -> "Projector is running"
+      isProjectorRunning() -> {
+        "Projector is running" //. Connected client(s): ${clientsCount()}"
```

Comment:
As we've decided not to implement connected clients count for now, I think it's better to revert such changes and add them in the next PR.

Reply:
Fixed